CHANGES.md

```diff
@@ -1,3 +1,8 @@
+## edge-18.11.1
+
+* Web UI
+  * **Improved** Fixed a smattering of small UI issues
```

Comment:
Question for @olix0r -- are there any user-facing proxy changes that we should mention? This release contains the following set of new proxy commits:

```
f97239b Use the grpc-status response header for classification (#113)
81b8378 Prepare HTTP metrics for per-route classification (#112)
4e0a1f0 refactor: Build stacks from bottom-to-top (#111)
70d2f57 Make `resolve` a stack module (#106)
af4dd6f Use rust:1.30.0 in development builds (#110)
6f031bd Introduce an optional fixed backoff between reconnects (#105)
66b1731 include address in TCP connect error messages
68ec7f1 fix test import conflict of assert_contains macro
2109f37 Make a generic Watch stack (#104)
4625302 Remove panic when failing to get remote address (#107)
f1210b4 Decouple reconnect & connect layers from proxy::http::client (#101)
978fed1 refactor: Structure the proxy in terms of `Stack` (#100)
```

Reply:
I think at least mentioning the change from linkerd/linkerd2-proxy#107 would be good.

Reply:
66b1731 is a logging change that will help us debug some open issues

4625302 resolves an issue that was reported recently

Reply:
https://github.com/linkerd/linkerd2/issues/1787